include/sway/commands.h

```diff
@@ -75,7 +75,8 @@ struct cmd_results *config_commands_command(char *exec);
 /**
  * Allocates a cmd_results object.
  */
-struct cmd_results *cmd_results_new(enum cmd_status status, const char* input, const char *error, ...);
+struct cmd_results *cmd_results_new(enum cmd_status status,
+		const char *error, ...);
```

Comment:
Style: I think we don't hit the 80 char limit if we put this on one line

Reply:
85.

Reply:
Fixing anyway -- the 80 char limit is soft, and the line originally was 103.